messaging/integration_test/src/integration_test.cc

```diff
@@ -222,7 +223,7 @@ bool FirebaseMessagingTest::CreateTestMessage(
     // Don't send HTTP requests in stub mode.
     return false;
   }
-  if (strcmp(kFcmServerKey, "REPLACE_WITH_YOUR_SERVER_KEY") == 0) {
+  if (strcasecmp(kFcmServerKey, "replace_with_your_server_key") == 0) {
```

Comment:
Same here

Reply:
I modified this one because the restore_secrets script was replacing it, although in this part of the code it should still have the placeholder value. (This prints a warning if it hasn't been modified.) The other one was in error and has been reverted.